apps/server/src/infra/database/management/database-management.service.ts

```diff
@@ -1,16 +1,17 @@
 import { MikroORM } from '@mikro-orm/core';
+import { Collection } from '@mikro-orm/mongodb/node_modules/mongodb';
+import { MigrateOptions, UmzugMigration } from '@mikro-orm/migrations-mongodb';
 import { EntityManager } from '@mikro-orm/mongodb';
 import { Injectable } from '@nestjs/common';
 import { BaseEntity } from '@shared/domain/entity';
-import { Collection, Db } from 'mongodb';
-import { MigrateOptions, UmzugMigration } from '@mikro-orm/migrations-mongodb';
 
 @Injectable()
 export class DatabaseManagementService {
 	constructor(private em: EntityManager, private readonly orm: MikroORM) {}
 
-	private get db(): Db {
-		return this.em.getConnection('write').getDb();
+	private get db() {
```

Comment:
Is always a good idea to expose the return type. Why was it removed?

Reply:
Because the different versions of mongodb caused errors here and we don't get the type through @mikro-orm/mongodb. But it was before I had the idea to import it like this: @mikro-orm/mongodb/node_modules/mongodb